inc/class-cachify.php

```diff
@@ -578,8 +578,12 @@ public static function add_dashboard_count( $items = array() ) {
 				),
 				admin_url( 'options-general.php' )
 			),
-			esc_attr( strtolower( $method ) ),
-			esc_html__( 'Caching method', 'cachify' ),
+			sprintf(
+				/* translators: 1: "Caching method label"; 2: Actual method. */
+				esc_html__( '%1$s: %2$s', 'cachify' ),
+				esc_html__( 'Caching method', 'cachify' ),
+				esc_attr( strtolower( $method ) ),
```

Comment:
Removing the comma would help passing the tests against older versions.

Reply:
You are right, sorry :)